pkg/serverless/logs/logs.go

```diff
@@ -305,7 +306,7 @@ func processLogMessages(c *LambdaLogsCollector, messages []logMessage) {
 		return messages[i].time.Before(messages[j].time)
 	})
 	for _, message := range messages {
-		processMessage(&message, c.ExecutionContext, c.EnhancedMetricsEnabled, c.ExtraTags.Tags, c.Demux, c.HandleRuntimeDone)
+		processMessage(&message, c.ExecutionContext, c.EnhancedMetricsEnabled, c.ExtraTags.Tags, c.Demux, c.InitDurationChan, c.HandleRuntimeDone)
```

Comment:
One thing I like to do when the number of arguments gets insane is to create an arg struct. For example,

```go
type MyArgs struct {
    message Message
    context Context
    purple  Purple
    happy   bool
}

func doSomethingAwesome(args MyArgs) error {
    if args.happy {
        return nil
    }
    return errors.New("I'm unhappy!")
}
```

Reply:
On second thought, why aren't we just doing

```go
func processMessage(msg *message, c * LambdaLogsCollector) { ... }
```

Reply:
Need to rebase this anyway